client/client/main.go

```diff
@@ -17,7 +21,7 @@ import (
 var lenSalt = flag.Int("len_salt", 8, "the length of the salts used to generate the PRFs")
 var fpRate = flag.Float64("fp_rate", 0.000001, "the desired false positive rate for searchable encryption")
 var numUniqWords = flag.Uint64("num_words", uint64(100000), "the expected number of unique words in all the documents within one TLF")
-var clientDirectories = flag.String("client_dirs", "/keybase/private/jiaxin,songgao;/keybase/private/jiaxin,strib", "the keybase directories for the client where the files should be indexed, separated by ';'")
+var clientDirectories = flag.String("client_dirs", "/keybase/private/test;/keybase/private/test,test2", "the keybase directories for the client where the files should be indexed, separated by ';'")
```

Comment:
Maybe this shouldn't have a default at all, since these defaults will break if anyone runs them.  And then error out if no clientDirectories are provided.

But if you don't have time today, it's fine as is.